packages/cli/src/oclif/commands/build.js

```diff
@@ -35,6 +36,11 @@ BuildCommand.flags = buildFlags({
         'Skips installing a fresh copy of npm dependencies on build. Helpful for using `yarn` or local copies of dependencies.',
       hidden: true,
     }),
+    'skip-validation': flags.boolean({
+      description:
+        "Skips a local and remote validation check of the CLI app's schema and AppVersion integrity.",
```

Comment:
In case anyone _is_ reading through the code, you might mention that this is skipping pre-push checks